### PR TITLE
CSS Grid Changes 

### PR DESCRIPTION
**Pull request by brian-penguin**
Before: 
Before

After: 
After

After- Print-view 
after-print-view

Hi Edward! 
I've been working on getting some CSS Grid understanding for some other projects at work for a client. I figured this is a pretty good application of them. Also added some font that I enjoy (feel free to change it)

Changes: 
- Format the HTML (editor did this autotmagically) 
- Add a link to the source code 
- Add CSS Grid 
- Change font sizes to use relative sizing with the base being the Category level set to size 14px

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,50 +8,51 @@
...
+          Leaders
+          Officer [15]
+          Sabre [3]
+          Squads
+          Skirmishers [3] x5
+          Rifles [1] x5
         </textarea>
         <div id="help">
           <a href="help.html" target="_blank">Open help page</a>
+          <a href="https://github.com/edward3h/simple_army_editor" target="_blank"> Source Code</a>
```
Comment (edward3h) [inline index.html]: I'd prefer something like "View source in Github"

**Review by edward3h — COMMENTED** [review]
I have a challenge: do you think you can make it so the user can select a font from a drop-down box? (I haven't yet thought about how that might work :-) I'm not entirely sure about defaulting to a monospace font.

Otherwise looks good, thanks!

**Comment by brian-penguin** [comment]
Update to allow font-family selections and persist between reloads / page visits using localStore

take2
take3

**Review by edward3h — APPROVED** [review]
Thanks for adding the font selector

**Pull request merged**